doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+ 
+
+# PULL REQUESTS
+## HOW TO CONTRIBUTE YOUR FIRST CODE (PR)
+Please see [Step By Step Guide To Contributing on GitHub](https://www.dataschool.io/how-to-contribute-on-github/) - This link provides useful information on creating your code contribution, submitting the PR, discussing PR and the process that needs to be followed. Make sure that you are following the criteria outlined below in these guidelines.
+
+The following resources may help you understand the process before making Pull Requests.
+* [How TO USE THE PULL REQUEST VIDEO EXPLAINATION](https://youtu.be/rgbCcBNZcdQ) 
+
+* Using the `Pull Request Template` will help to guide you in making useful and neat contributions.
+
+### NAMING BRANCHES
+For naming your new branches, it is good practice to consider naming the branch in your name or by the name of the problem being solved. Also link it with the number of the issue i.e. Jane/Nav-bar/12 
+<yourName>/<featureName>/<issueNumber>
+ 
+### NAMING THE PULL REQUEST
+When deciding a title/name for your pull request please prefix with:
+
++ (Feat): For a new feature e.g. if I were creating a new Navbar the title would be (Feat): This is a new feature and the new feature is a Navbar, but the title should be something like '(Feat): Add a Navbar component'.
++ (Fix): I am fixing something.
++ (Doc): Some new documentation.
+I.e. "fix.folder-structure" this outlines what you are doing and what you have done. It is short and sweet, to the point and informative. It will make documenting and revisiting code in the future easy to read and understand.
+ 
+### APPROVALS BEFORE MERGING 
+It is important to seek approvals of your Pull Requests before merging your work to the main branch. It is good practice to obtain approvals from 3 or more team members. These will avoid reopening PR.
+ 
+### MERGES
+Once you have an approved PR, your contribution can be merged to the main branch and you can continue to work on a new Issue, repeating the process.
+
+# TO BE PRODUCTIVE, IN A RAPIDLY DEVELOPING ENVIRONMENT - 
+## PLEASE READ THE GOVERNANCE BELOW:
+ 
+### 1. Your first contribution?  
+Get excited you’re about to contribute to a great team project with your code.
+
+### 2. Meet each team member where they are and be super supportive
+- as we all strive for greatness, please see this wonderful cheat sheet to help you format your GitHub code properly. [CLICKE HERE](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+### 3. Allow team members to be spontaneous
+- expect unexpected changes and commits. Make sure to comment, comment, comment.
+
+### 4. Don't expect them “to know”. 
+- It's OK to be wrong, but always research for answers you need as these contribution guidelines are a way to communicate how people should contribute. 
+Also read [Code Of Conduct](https://github.com/tanyapowell/restaurant-reviews/blob/add-code-of-conduct-1/CODE_OF_CONDUCT.md)
```

Comment:
Probably best to remove this for now until you've finished #14. Once that's been merged you can point to it here